curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-linked-list-class.md

```diff
@@ -57,6 +57,7 @@ assert(
     var test = new LinkedList();
     test.add('cat');
     test.add('dog');
+    test.add('fish');
     return test.head().next.element === 'dog';
```

Comment:
What do you think about adding check if currently last (`next.next`) element is actually `fish`?